installer/conf/omsagent.d/baseline.conf

```diff
@@ -0,0 +1,30 @@
+# Baseline plugins
+
+<source> 
+    type exec
+    tag oms.baseline_immediate_run
+    command /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d 
+    format json
+</source>
+
+<source> 
+    type exec
+    tag oms.baseline

```

Comment:
Do you need separate tags for the immediate and regular runs?
If you don't you can combine the 2 filter plugins into one